packages/sdl2_image.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Sdl2_image < Package
+  description 'SDL2_image is an image loading library that is used with the SDL2 library.'
+  homepage 'https://www.libsdl.org/projects/SDL_image/'
+  version '2.0.4'
+  source_url 'https://www.libsdl.org/projects/SDL_image/release/SDL2_image-2.0.4.tar.gz'
+  source_sha256 'e74ec49c2402eb242fbfa16f2f43a19582a74c2eabfbfb873f00d4250038ceac'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'libsdl2'
+  depends_on 'libjpeg'
+  depends_on 'libpng'
+  depends_on 'libtiff'
+  depends_on 'libwebp'
```

Comment:
The `libwebp` package already includes `libpng` and `libtiff` dependencies.

Reply:
It's what, another five seconds to resolve the dependencies? Five seconds for the security of knowing that all the dependencies are installed.

Reply:
Apparently, you don't think it's a good idea to try and prevent possible dependency loops.

Reply:
I do, but this package compiles and runs correctly. There is no need to prevent "possible dependency loops" if the packages in question cannot even use `SDL2_image`.

Reply:
Just so you know, there's a difference between redundant dependencies and dependency loops.

Reply:
This is a pointless argument.  I will just remove after adding the pre-built binaries.

Reply:
Nice to know you take into account my opinion.

Reply:
Thank you for new package.

Reply:
You're welcome. I'm sorry that I... overreacted.